smtp/message.js

```diff
@@ -63,12 +97,13 @@ class Message {
 				header === 'attachment' &&
 				typeof headers[header] === 'object'
 			) {
-				if (Array.isArray(headers[header])) {
-					for (let i = 0, l = headers[header].length; i < l; i++) {
-						this.attach(headers[header][i]);
+				const h = headers[header];
```

Comment:
in general, i prefer to not have single letter variable names. what about renaming this `header`? i've seen similar changes like `m` for `message`.

i use to do this as well because it would shave off characters for the web, but now with modern tooling or running on the server side, this practice isn't necessary (unless you are in a for loop, that seems to be standard)

Reply:
renamed to `attachment`, since that's the property we're accessing